src/widgets/search-box/search-box.js

```diff
@@ -268,11 +269,6 @@ export default function searchBox({
   }
 }
 
-// the 'input' event is triggered when the input value changes
-// in any case: typing, copy pasting with mouse..
-// 'onpropertychange' is the IE8 alternative until we support IE8
-// but it's flawed: http://help.dottoro.com/ljhxklln.php
```

Comment:
comment was deleted, but not the code that triggered the comment :)

Reply:
Done!